MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -451,6 +483,7 @@ + (void)setCustomerData:(nullable MUXSDKCustomerData *)customerData forPlayer:(n
     MUXSDKCustomerPlayerData *playerData = [customerData customerPlayerData];
     MUXSDKCustomerViewData *viewData = [customerData customerViewData];
     MUXSDKCustomerVideoData *videoData = [customerData customerVideoData];
+    MUXSDKCustomData *customData = [customerData customData];
 
     if (!playerData && !videoData && !viewData) return;
```

Comment:
and here

Reply:
i've actually flipped this around in my other PR, there will be a fair amount of conflicts between these two 😩

Reply:
:(